### ISSUE TITLE
BUG: CSV Export history should ONLY show exports from the logged in user

### ISSUE THREAD

**Issue by rowasc** [ushahidi/platform#2601]
### Expected behaviour
- When I see my exports history, I only see the ones that belong to me
### Actual behaviour
- When I see my exports history as an admin, I see all the exports

### Steps to reproduce the behavior/error
- Login to http://csvexportsforever.v3-qa.ush.zone/settings/data-export or another qa deployment from v3-qa.ush.zone
- Go to the exports page
- Go to the history tab
- Expected behavior: You should only see your exports
- Current behavior: You see all exports

##### Where
- [x] [Local setup with Vagrant ](https://www.ushahidi.com/support/install-ushahidi#installing-for-development)
- [ ] [Local setup from platform-release ](https://www.ushahidi.com/support/install-ushahidi#installing-the-latest-release)
- [ ] Ushahidi.io / SaaS solution 
- [x] Ushahidi's QA environment (in v3-qa.ush.zone) 
- [ ] Other (explain):

**Comment by rowasc** [comment]
@Angamanga I couldn't reproduce this.

**Comment by Angamanga** [comment]
@rowasc @caharding Unfortunately, yes I can reproduce it on .io. I created a new user and went to the data-export in settings and started an export. In the export-tab I see all export-jobs that is finished. 

I also see all the export-jobs in the response from the api so its not a caching-issue.

**Comment by willdoran** [comment]
### Spec:

- [ ] Export Job History should show only export jobs from the currently logged in user
- [ ] Export Job Repository should filter export jobs by current user

I believe the reason it was like this is because in order to perform an Export you must be an admin and by default all admins can see everything.

@caharding @Angamanga Instead of restricting the jobs to the current user would it be better to add information to the Job showing which user started the job:
'Created by X'

**Comment by Angamanga** [comment]
@willdoran As long as its only admins that can see other users exports thats ok i think.

**Comment by Angamanga** [comment]
@willdoran Looks good, moving to ready for dev!

**Comment by jrtricafort** [comment]
Needs a new spec. @willdoran

**Comment by rowasc** [comment]
@jrtricafort do you remember why this was flagged as "needs a new spec" ?

**Comment by jrtricafort** [comment]
@rowasc That comment was to capturing something from a conversation I had with @willdoran , but I don't recall the specifics. Will do you remember? If not, I think we can probably ignore that for now, since we have prioritized this as P2 so not as high a priority.

**Comment by rowasc** [comment]
Thanks @jrtricafort.  I'd prefer to move this to cycle 7 if it's really a P2 for us , since cycle 6 is pretty full at the moment and we're even behind on specs :/

**Comment by jrtricafort** [comment]
@rowasc Done

**Comment by crcommons** [comment]
@willdoran Do you recall your conversation with Juan about why this needed new spec?

**Comment by willdoran** [comment]
@crcommons I think though I failed to write this down it's because admin users should not see all jobs but only their own. But may be double check that with @jrtricafort

**Comment by jrtricafort** [comment]
@willdoran @crcommons That's correct. Admin users should only see their jobs.

There's some ambiguity here, because we do tend to give admin users lots of superpowers over lots of their own deployments. But seeing all exports with the username (as was originally spec'd for this issue back in May) is more like an audit of other users' behavior, and at no other point in the UX do we support that. If there is ever a use-case where someone actually needs to see who is exporting what, that is more of a privacy workflow and that should be treated as a separate special thing (which doesn't currently exist outside of hitting up the DB directly on the backend).

So long story short, yep, even admins should just see their own exports in export history.

**Comment by crcommons** [comment]
@jrtricafort I've implemented this fix in develop, however, with Kohana (which is in production) we have it so that users who have all manage settings can't even access exports at all, let alone their own exports. Is this something we need to change now in production or can we wait until lumen is released?

Screenshot shows a logged in user with manage settings, posts, and users. Clearly this user cannot do those things.

![Screen Shot 2018-07-17 at 8.55.17 AM.png](https://images.zenhubusercontent.com/5a7084ec4b5806bc2bc4e02b/a8b496cc-8ad9-4f74-bdb0-79898e18f5dd)

**Comment by rowasc** [comment]
@jrtricafort @crcommons I added an issue about Manage Settings not allowing some of the permissions it states in the docs that we allow. https://github.com/ushahidi/platform/issues/3169 
Manage settings is not supposed to allow a CSV download as far as I know but I realized that we also are not giving users with Manage Settings the permission to see / edit categories .

**Comment by crcommons** [comment]
UPDATE:

- Changes in master have been merged in client and API and waiting to deploy;
- changes in develop have been merged in API and waiting to deploy;
- changes in develop in client are *waiting for review*

**Comment by crcommons** [comment]
Update: changes in client have been merged into develop.

**Comment by Eve-wanderi** [comment]
@rowasc  This has passed for QA.

![image](https://user-images.githubusercontent.com/36038029/43203257-69f06edc-9026-11e8-9d53-0053fd0f03ec.png)

![image](https://user-images.githubusercontent.com/36038029/43202854-41ccec60-9025-11e8-9b2a-a90b6aadabd8.png)

**Comment by Eve-wanderi** [comment]
Confirmed in http://targetedsurveytesting.v3-qa.ush.zone/ works as expected.
